frontend/src/components/formInput/index.js

```diff
@@ -51,15 +51,14 @@ class FormInput extends Component {
   }
 
   render() {
-    const { name, className, errors, password } = this.props
+    const { name, className, errors, type } = this.props
     const { id, toolTipId } = this
     const { passwordVisible } = this.state
     const hasErrors = errors != 'loading' && errors && errors.length > 0,
           iconClassName = ((errors == null || errors == 'loading') && 'noClass') || (errors.length <= 0 ? styles.iconCheck : styles.iconTimes),
           iconName = (errors == null && 'MinusCircle') || (errors == 'loading' && 'Cog') || (errors.length <= 0 ? 'CheckCircle' : 'TimesCircle'),
-          inputType = password && !passwordVisible ? 'password' : 'text',
           loadingClass = errors == 'loading' ? 'fa-spin' : '',
-          isPassword = password
+          isPassword = type == "password"
```

Comment:
Please check the form. Clearly states usage of single-quotes

```suggestion
          isPassword = type == 'password'
```